tools/mcsema_lift/Lift.cpp

```diff
@@ -37,16 +37,16 @@
 #include "mcsema/BC/Util.h"
 
 #ifndef LLVM_VERSION_STRING
-# define LLVM_VERSION_STRING LLVM_VERSION_MAJOR << "." << LLVM_VERSION_MINOR
+#define LLVM_VERSION_STRING LLVM_VERSION_MAJOR << "." << LLVM_VERSION_MINOR
```

Comment:
Was it a clang format that removed those spaces? The indentation in the `#` macro things was intentional.

Reply:
It was, yea, I have clang-format set to run on save in my IDE, but I can go back and undo this, if it is misbehaving.